libs/service-portal/settings/src/components/UserOnboardingModal/UserOnboardingModal.tsx

```diff
@@ -63,9 +69,31 @@ const UserOnboardingModal: ServicePortalModuleComponent = ({ userInfo }) => {
     }
   }
 
-  const handlePhoneStepSubmit = (data: PhoneFormData) => {
+  const handlePhoneStepSubmit = async (data: PhoneFormData) => {
     setTel(data.tel)
-    gotoStep('email-form')
+    try {
+      setButtonLoading(true)
+      // TODO: send sms to telephone
```

Comment:
The phone confirmation step should probably be sending the sms and verifying it within itself